lib/assessment.rb

```diff
@@ -5,7 +5,7 @@ class Assessment
   # enumerable supplied.  Assume each entry implements a length
   # method that returns a Fixnum.
   def longest(enum)
-    # Your code goes here
+    enum.max_by { |word| word.length }
```

Comment:
Correct! We also would have accepted 
`enum.max { |a, b| a.length <=> b.length }`